src/Sentry.Maui/Internal/MauiDeviceData.cs

```diff
@@ -6,29 +6,22 @@ namespace Sentry.Maui.Internal;
 
 internal static class MauiDeviceData
 {
-    public static void ApplyMauiDeviceData(this Device device, IDiagnosticLogger? logger)
+    public static void ApplyMauiDeviceData(this Device device, IDiagnosticLogger? logger, INetworkStatusListener? networkStatusListener, string deviceName, string deviceManufacturer, string deviceModel, string deviceType, bool? deviceSimulator, DevicePlatform devicePlatform, DisplayInfo displayInfo, bool supportsVibration, bool supportsAccelerometer, bool supportsGyroscope)
```

Comment:
This is not the best when it comes to readability, and I'm not sure it makes much sense. (it works as is but it does mean it's the ultimate way to go)

so I need to find a better way